compact/ecal/backward_PbWO4.xml

```diff
@@ -110,7 +110,7 @@
             <position x="7.175*cm"  y="7.175*cm"/>
             <position x="7.175*cm"  y="5.125*cm"/>
             <position x="9.225*cm"  y="1.025*cm"/>
-            <position x="9.225*cm"  y="3.075*cm"/>
+            <position x="9.225*cm"  y="3.075*cm"/>           
             <position x="-1.025*cm" y="7.175*cm"/>
             <position x="-1.025*cm" y="9.225*cm"/>
             <position x="-3.025*cm" y="7.175*cm"/>
```

Comment:
The problem is this value..... it supposed to be -3.075*cm, and it became -3.025*cm during copy paste..... it is totally eye test :)

Reply:
I will fix this in my next commit because I want to see is there any other overlapped issue appears in my last commit.

Reply:
This is of course *exactly* why we don't want to have to maintain these long lists of module positions in a file but prefer to calculate them directly in the code...

Reply:
I totally agree and I just suffered from that...... I will try to think about how to fill the box in the 12-sides polygon by calculation but I think it is fine to fill the gap between inner side and beam pipe by hardcoded, as number of modules are relatively less and there might have new design of supporting structure. I will add some comments in xml file and put 1 or 2 line for spacing inner and outer additional modules.

Reply:
correct the values in the last commit